sympy/core/tests/test_var.py

```diff
@@ -1,83 +1,91 @@
 # Tests for var are in their own file, because var pollutes global namespace.
-
+import textwrap
 from sympy import Symbol, var, Function, FunctionClass
 from sympy.utilities.pytest import raises
 
 # make z1 with call-depth = 1
 
 
-def _make_z1():
-    var("z1")
 
 # make z2 with call-depth = 2
```

Comment:
The same as for the previous comment. The comment `# make z2 with call-depth = 2` belongs to the function `_make_z2` so should be near it.

Reply:
Fine. Will do the necessary changes immediately.

Reply:
Shall I do it immediately, as then it will restart the TRAVIS CI build?